docs/releases/1.11.txt

```diff
@@ -151,6 +151,11 @@ Minor features
 * Added the ability to pass the `size`, `shape`, and `offset` parameter when
   creating :class:`~django.contrib.gis.gdal.GDALRaster` objects.
 
+* Added support for the
```

Comment:
I would say "Added SpatiaLite support for the IsValid function, MakeValid function, and isvalid lookup."